tensorrt_llm/_torch/auto_deploy/transform/library/multi_stream_moe.py

```
@@ -306,20 +415,23 @@ def _apply(
         cuda_stream_manager.add_device(torch.cuda.current_device())
-        gm, num_matches = _execute_op_in_aux_stream(gm, op_dict)
+        gm, num_matches = _execute_shared_expert_in_aux_stream(gm, base_ops)
 
         info = TransformInfo(
             skipped=False,
             num_matches=num_matches,
             is_clean=num_matches == 0,
             has_valid_shapes=num_matches == 0,
         )
-
+        with open("/tmp/after_multi_stream_moe.txt", "w") as f:
```

Ruff (0.15.0)

[error] Probable insecure usage of temporary file or directory: "/tmp/after_multi_stream_moe.txt"

(S108)